packages/edit-post/src/editor.js

```diff
@@ -180,28 +173,24 @@ function Editor( {
 	}
 
 	return (
-		<StrictMode>
```

Comment:
We should keep in mind to open a PR enabling `StrictMode` as soon as we land this one.

Reply:
Strict mode enabler PR opened here: #47639